src/containers/my-courses/courses-filters-drawer/CoursesFiltersDrawer.tsx

```diff
@@ -61,17 +93,19 @@ const CoursesFiltersDrawer: FC<CoursesFiltersDrawerProps> = ({
       <Box sx={styles.categorySelect}>
         <Typography sx={styles.titleMargin}>
           {t('myCoursesPage.coursesFilter.chooseThe')}
-          <Typography sx={styles.boldText}>
+          <Typography component={'span'} sx={styles.boldText}>
```

Comment:
try to use `ComponentEnum` for `span`
Check please other your properties `component={'span'}` and replace it with `enum`

Reply:
thanks! fixed